### DISCUSSION TITLE
Use GPIO buttons also for YES and NO

### DISCUSSION THREAD

**Discussion by baumrasen** [photobooth-app/photobooth-app#279] [Ideas]
A would like to use the GPIO buttons also for "YES" and "NO" action. So it would be possible, to react and "say NO" without the touchscreen.

**Comment by mgineer85** [comment]
#250 not yet 200% thought trough so not yet implemented

**Reply by baumrasen** [reply]
Thanks for the link.
I like your response time. This is amazing! ❤️
And I like your app. It is working great. ❤️❤️❤️

Maybe, I will have some enhancement requests 😊
